receiver/hostmetricsreceiver/metadata.yaml

```
@@ -293,6 +293,22 @@ metrics:
       monotonic: false
     labels: [network.protocol, network.state]
 
+  system.network.conntrack.count:
+    description: Number of currently allocated flow entries.
+    unit: 1
```

Something like
```suggestion
    unit: "{flow entries}"
```
might be more useful